tests/cross_functional/conftest.py

```diff
@@ -1591,3 +1591,64 @@ def factory(
         logger.info(f"No {search_string} errors are found in the noobaa pod logs")
 
     return factory
+
+
+@pytest.fixture(scope="session")
+def setup_stress_testing_bucket(bucket_factory_session, rgw_bucket_factory_session):
+    """
+    This session scoped fixture is for setting up the buckets for the stress testing
+    in MCG. This creates buckets of type AWS, AZURE, PV-POOL, RGW.
+
+    """
+
+    def factory():
```

Comment:
Please add a docstring for declaring the structure of the dict this factory returns